client/api/planning.ts

```diff
@@ -200,6 +200,38 @@ function setDefaultValues(
     );
 }
 
+function bulkAddCoverageToWorkflow(planningItems: Array<IPlanningItem>): Array<Promise<IPlanningItem>> {
+    const {getState, dispatch} = planningApi.redux.store;
+    const {gettext} = superdeskApi.localization;
+    const {notify} = superdeskApi.ui;
+
+    const coverageStatuses = selectors.general.newsCoverageStatus(getState());
+    const planningItemsToUpdate: Array<IPlanningItem> = planningItems.filter((item) => item.lock_action !== 'edit');
+
+    return planningItemsToUpdate.map((plan) => {
+        const updates = {coverages: cloneDeep(plan.coverages)};
+
+        updates.coverages = plan.coverages
+            .map((coverage) => planningUtils.getActiveCoverage(coverage, coverageStatuses));
+
+        return planning.update(plan, updates)
+            .then((updatedPlan) => {
+                notify.success(gettext('Coverage added to workflow.'));
```

Comment:
I wonder if we should show it per item, that could be a lot of notifications, might be good to do some summary at the end